web/profiles/webspark/webspark/webspark.install

```diff
@@ -453,7 +453,28 @@ function webspark_update_10001(&$sandbox) {
  */
 function webspark_update_10002(&$sandbox) {
   \Drupal::state()->set('configuration_locked', FALSE);
-  \Drupal::service('webspark.config_manager')->updateConfigFile('editor.editor.basic_html');
-  \Drupal::service('webspark.config_manager')->updateConfigFile('filter.format.basic_html');
+  $config_factory = \Drupal::service('config.factory');
+  $editor_cofigs = $config_factory->listAll($prefix = 'editor.editor');
```

Comment:
Typo: can we use `$editor_configs` instead of `$editor_cofigs` ?

It's minor, but I like to keep typos even out of variable names, even when they're used accurately. :)